### PR TITLE
Localbackend tests

### PR DESCRIPTION
**Pull request by clux**
Based off #4 

Trying to rewrite tests. WIP.

### CODE REVIEW

File: configs/demo.json [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-      "master": "https://artifactory.host",
-      "slave": "https://artifactory.host",
-      "release": "DEMO-release",
-      "vgroup": "DEMO-group",
-      "credentials": null
-    }
+    "local": {}
   },
   "environments": {
+    "alp36": {
```
Comment (bencord0) [inline configs/demo.json]: "alpine36" or "alpine".
Is there a necessity for this specific tag? What do we do about upgrades?

File: .gitignore [hunk 2]
```diff
@@ -7,4 +7,8 @@ lal.tar
 ARTIFACT
 tests/helloworld/INPUT
 tests/helloworld/.lal
+tests/heylib/INPUT
```
Comment (bencord0) [inline .gitignore]: Maybe move the `tests/` ignores to `tests/.gitignore`.

File: configs/demo.json (hunk 1 above)
Comment (clux) [reply configs/demo.json]: I'll expand it, sure.

Not entirely sure if we need to be this specific either. It was intended as an abi safety by tag, but as long as the app at the top is at least newer, then we should be good. At least that was the case for ubuntu's. Alpine doesn't use glibc I guess, so maybe we can just use libraries across different alpines in newer and older alpine containers.

**Comment by clux** [comment]
travis ran into an alpine image issue https://github.com/lalbuild/dockerfiles/issues/1
fixing now

**Comment by clux** [comment]
It's all green. Merging :-)

**Comment by clux** [comment]
localbackend released in lal@3.8.0

**Pull request merged**